### DISCUSSION TITLE
Dynamic duration

### DISCUSSION THREAD

**Discussion by ChristianP85** [PyVRP/PyVRP#661] [Q&A] [answered]
Hi,
Is it possible to dynamically change the duration between nodes? For example, if a road closure at a certain time needs to be considered, or if there is more traffic on certain routes in the afternoon than in the morning?

**Comment by N-Wouda** [comment, marked as answer]
Hi @ChristianP85, what you describe is a time-dependent routing problem. We do not currently support this, but it is tracked here: https://github.com/PyVRP/PyVRP/issues/441. We might get around to supporting this eventually, but that is currently not planned.
